packages/browser/src/index.ts

```diff
@@ -93,14 +98,52 @@ export default class LogtoClient {
   }
 
   protected set signInSession(logtoSignInSessionItem: Optional<LogtoSignInSessionItem>) {
```

Comment:
shall we align the usage of `Optional` and `Nullable` across the props?

Reply:
Aligned with `Nullable` here. PTAL